assets/tests/userTestGroups.test.ts

```diff
@@ -1,24 +1,64 @@
+import { StructError } from "superstruct"
 import appData from "../src/appData"
 import inTestGroup from "../src/userTestGroups"
 
 jest.mock("appData")
 
-describe("User test groups helper", () => {
-  // test("Assert")
-  test("Assert detect user in group", () => {
+describe("User Test Groups API", () => {
+  test("raise error when given wrong json data type", () => {
+    ;(appData as jest.Mock)
+      .mockImplementationOnce(() => ({
+        // Wrong data type
+        userTestGroups: null,
+      }))
+      .mockImplementationOnce(() => ({
+        // Wrong data type
+        userTestGroups: "1.0",
+      }))
+      .mockImplementation(() => ({
+        // Wrong data type
+        userTestGroups: "{}",
+      }))
+
+    // Start with no groups or keys
+    expect(() => inTestGroup("non-existent-group")).toThrow(StructError)
+    expect(() => inTestGroup("non-existent-group")).toThrow(StructError)
+    expect(() => inTestGroup("non-existent-group")).toThrow(StructError)
+  })
+
+  test("raise error when given invalid json", () => {
```

Comment:
Sorry, what I meant to get at with my earlier comment wasn't only cases where the value is something that doesn't parse as JSON, but also when it's something like `"{"key": "value"}"` that is valid JSON, but not an array of strings when parsed.